content/applications/sales/subscriptions/closing.rst

```diff
@@ -1,105 +1,114 @@
+===================
+Close subscriptions
+===================
 
-Losing a customer is always difficult, especially if you put a lot of effort into getting them to
-sign up for your products/services. However, many companies come up with dubious methods to reduce
-the probability of this happening.
+While losing customers is not ideal, making it easy for customers to cancel subscriptions minimizes
+the impact of losing them, fostering a positive experience for the customer *and* the business.
 
-.. raw:: html
+If the subscription closing process is made unnecessarily difficult or cumbersome, that will likely
+increase the level dissatisfaction amongst the leaving customer base, which can negatively impact
+the reputation of the company.
 
-   <div align="center" style="color:#AD5E99; font-size: 2rem ;margin: 20px 0"> <b>What is the right
-   way to go?</b> </div>
+By granting customers the power to close their own subscriptions, they do not feel "trapped" and are
+more likely patronize the same business again in the future for other products, services, and/or
+subscriptions.
 
-1. If you adopt such methods, you are among companies that actively and indirectly spread the
-   phobia of subscriptions, who do not think about how negatively dissatisfied customers could impact
-   their business and, who frustrate the customers in a way or another. However, at some point, it is
-   understandable that you do not want your customers to be involved in your subscription status.
-
-2. If you do not adopt such methods, you are among companies that continue to evolve in a
-   long-term subscription business model, especially in an era of ultra-fast communications between
-   dissatisfied customers, and who retain their customers by making it easier for them to leave if
-   they want to. By giving them the opportunity to close their own subscriptions, your customers do
-   not feel trapped because they subscribed to your products/services.
-
-In summary, the **Odoo Subscriptions** application gives you the
-**possibility to choose what you want to apply**. Indeed, you can decide whether to give your customers the
-option to close their subscriptions whenever they want to or to restrict this possibility. It depends
-on you and we will show you how to do that in our amazing application.
+Odoo *Subscriptions* provides flexibility for businesses to decide whether customers can self-manage
+their subscriptions, or restrict that ability entirely.
 
 Configuration
 =============
 
-Go to :menuselection:`Subscriptions --> Configuration --> Subscription templates`. From there, you can
-create a new *Subscription template* or modify an existing one. When editing your template, underneath
-the Invoicing tab, you have the possibility to activate the option *Closable by customer*.
-Once enabled, this option gives your customers the right to close their own subscriptions.
+To configure recurring plans offered, navigate to :menuselection:`Subscriptions app -->
```

Comment:
First part not needed or seems forced. Suggestion to just tell the reader to go to the right dash.
```suggestion
Start by navigating to :menuselection:`Subscriptions app -->
```